src/ui/server/controllers/insurance/business/turnover/alternative-currency/index.ts

```diff
@@ -131,6 +133,22 @@ export const post = async (req: Request, res: Response) => {
       return res.redirect(PROBLEM_WITH_SERVICE);
     }
 
+    /**
+     * If is a change route
+     * redirect to TURNOVER_CHANGE
+     */
+    if (isChangeRoute(req.originalUrl)) {
+      return res.redirect(`${INSURANCE_ROOT}/${referenceNumber}${TURNOVER_CHANGE}`);
+    }
+
+    /**
+     * If is a check-and-change route
+     * redirect to TURNOVER_CHECK_AND_CHANGE
+     */
+    if (isCheckAndChangeRoute(req.originalUrl)) {
+      return res.redirect(`${INSURANCE_ROOT}/${referenceNumber}${TURNOVER_CHECK_AND_CHANGE}`);
+    }
+
```

Comment:
Do we need to update the PR title/description to mention adding check/change answers instead of pure bug fixes?

Reply:
technically this is a bug :P